spylon_kernel/scala_magic.py

```diff
@@ -190,21 +230,38 @@ def trim(prefix, completions):
 
         prefix = info['code'][info['start']:info['help_pos']]
 
-        offset = trim(prefix, c)
+        offset = trim(prefix, completions)
 
-        a = [prefix + h[offset:] for h in c]
-        self.kernel.log.debug("info %s\n    completions %s\n     final %s", info, c, a)
-        return a
+        final_completions = [prefix + h[offset:] for h in completions]
+        self.kernel.log.debug('''info %s
+            completions %s
+            final %s''', info, completions, final_completions)
```

Comment:
Do you need to dedent this? I think this will render as:

```
info <info>
            completions <completions>
            final <final>
```

but not 100% certain